internal/services/reservations_service.go

```diff
@@ -52,3 +52,32 @@ func ListReservations(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+func GetReservation(w http.ResponseWriter, r *http.Request) {
```

Comment:
What about naming it with the status? smthing like `GetReservaionStatus`

Reply:
Basically, this function returns a reservation by id, not just the status

Reply:
No status please, kill status, let's be REST :-)